apps/web/src/app/events/[slug]/page.tsx

```diff
@@ -96,32 +96,35 @@ export default async function Page({ params }: { params: { slug: string } }) {
           style={{
             backgroundImage: `url('${event.coverImage}')`,
           }}
-        ></div>
+        />
 
         <div
           className={styles.backgroundColor}
           style={{
             background: `linear-gradient(to right, ${event.primaryColor}, ${colors.primary})`,
           }}
         />
-        {isFuture&&<div className={styles.containerWrapper} style={{ zIndex: 3 }}>
-          <div className={styles.container}>
-            <div className={styles.actionBox}>
-              <BasicCallendarButton
-                name={event.title}
-                startTime={event.from}
-                endTime={event.to}
-                timeZone="Europe/Warsaw"
-                location={
-                  event.address
-                    ? `${event.address.city} ${event.address.place}`
-                    : "online"
-                }
-                className={styles.btn}
-              />
+
+        {!isFuture && (
```

Comment:
@JUtupe ten "!" jest tutaj potrzebny? jak będzie `isFuture=false` to pojawi się przycisk do dodania kalendarza w innym wypadku no ni, a chyba feature ten dla przyszlych wydarzen mial być, czy ja tu coś teraz mieszam xd